src/foam/nanos/auth/Relationships.js

```diff
@@ -49,7 +49,7 @@ foam.RELATIONSHIP({
   forwardName: 'notificationSettings',
   inverseName: 'owner',
   sourceProperty: {
-    hidden: true
+    section: 'systemInformation'
```

Comment:
We don't want this hidden anymore?

Reply:
i was trying to show as many properties as possible .. and then we can hide those properties by hiding the entire section. I am showing these notification settings in case the system administrator wants to see them in the future.